### PR TITLE
Propose genesis file for testnet

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: genesis.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "config": {
+      "chainId": 418548838258,
```
Comment (antsankov) [inline genesis.json:3]: Can we use something more sane for the chainID? Feel free to chose a number or we can do either 212, or 545. I like them both.

File: genesis.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    },
+    "nonce": "0x42",
+    "timestamp": "0x0",
+    "extraData": "0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa",
+    "gasLimit": "0x1fffffffffffff",
+    "difficulty": "0x10000000000",
+    "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "alloc": {
+      "6e3da3dd22f043958fdb862db5876e5e52a3d7b7": {
```
Comment (antsankov) [inline genesis.json:16]: Here are the addresses I will use for testing, if you an add those in:

0x2958DB51a0b4c458d0aa183E8cFB4f2E95cf6E75
0xB460ce2C10d959251792D72b7E2B3EC684F013f1
0x25848A733da7c782fB963D2e562cF7246bD5b6df

**Review by antsankov — COMMENTED** [review]
Looks good, should be ready to launch testnet soon.

File: genesis.json (hunk 1 above)
Comment (atoulme) [reply genesis.json:3]: Both seem free: https://chainid.network/ I'll go with 212.

Comment (atoulme) [reply genesis.json]: Done

File: genesis.json (hunk 2 above)
Comment (atoulme) [reply genesis.json:16]: Added.

**Review by antsankov — APPROVED** [review]

**Pull request merged**